packages/designmanual/stories/molecules/NotionBlock.tsx

```diff
@@ -7,159 +7,246 @@
  */
 
 import React from 'react';
-
+import styled from '@emotion/styled';
 import { ConceptNotion } from '@ndla/ui';
 // @ts-ignore
 import { initArticleScripts } from '@ndla/article-scripts';
 import { uuid } from '@ndla/util';
+import { breakpoints, mq } from '@ndla/core';
 //@ts-ignore
 import { useRunOnlyOnce } from '../article/useRunOnlyOnce';
 
+const ContentWrapper = styled.div<{ inSearchResults?: boolean }>`
+  position: relative !important;
+  right: auto !important;
```

Comment:
Må det være important her? Er ikkje veldig glad i det for det tyder på css som ikkje heng helt sammen.